CitesphereConnector.py

```diff
@@ -70,6 +69,10 @@ def get_group_items(self, zotero_group_id):
     def get_collections(self, zotero_group_id):
         url = self.api + "/v1/groups/{}/collections".format(zotero_group_id)
         return self.execute_command(url)
+    
+    def get_collection_items_pg(self, zotero_group_id, collection_id,pg_number):
```

Comment:
this method can be folded into `get_collection_items`, just provide a default value for `pg_number` (which should be `page_number`) and only append the parameter to the url if it is not None.

Reply:
created one function for fetching collection items